spec/controllers/users_controller_spec.rb

```diff
@@ -39,13 +39,6 @@
   end
 
   context "new user is not created" do
-    it "response is successful" do
```

Comment:
Why was this removed? Alex made a comment here asking whether the response should be successfull i.e. 200. He did not say to remove the test. There is something incorrect in your implementation that should be changed and reflected in this test.